sdk/keyvault/azure-security-keyvault-certificates/test/ut/certificate_client_test.cpp

```diff
@@ -189,3 +187,64 @@ TEST_F(KeyVaultCertificateClientTest, GetCertificateVersion)
     EXPECT_NE(cert.Cer.size(), 0);
   }
 }
+
+TEST_F(KeyVaultCertificateClientTest, SetContacts)
+{
+  auto const& client
+      = GetClientForTest(::testing::UnitTest::GetInstance()->current_test_info()->name());
+
+  std::vector<CertificateContact> contacts;
+
+  CertificateContact ctt;
+
+  ctt.EmailAddress = "one@two.org";
+  ctt.Name = "giqu"; // cspell:disable-line
+  ctt.Phone = "1234567890";
+  contacts.emplace_back(ctt);
+
+  CertificateContact ctt2;
+
+  ctt2.EmailAddress = "two@three.org";
+  ctt2.Name = "giqu2"; // cspell:disable-line
+  ctt2.Phone = "1234567891";
+  contacts.emplace_back(ctt2);
+
+  auto response = client.SetContacts(contacts);
+
+  CheckContactsCollections(contacts, response.Value);
+
+  auto response2 = client.DeleteContacts();
+
+  CheckContactsCollections(contacts, response2.Value);
+}
+
+TEST_F(KeyVaultCertificateClientTest, GetContacts)
+{
+  auto const& client
+      = GetClientForTest(::testing::UnitTest::GetInstance()->current_test_info()->name());
+
+  std::vector<CertificateContact> contacts;
+
+  CertificateContact ctt;
+
+  ctt.EmailAddress = "one@two.org";
+  ctt.Name = "giqu"; // cspell:disable-line
+  ctt.Phone = "1234567890";
+  contacts.emplace_back(ctt);
+
+  CertificateContact ctt2;
+
+  ctt2.EmailAddress = "two@three.org";
+  ctt2.Name = "giqu2"; // cspell:disable-line
+  ctt2.Phone = "1234567891";
+  contacts.emplace_back(ctt2);
```

Comment:
Add tests combining the values we set for the contact. For example, in one contact, set only the email. In other set only the name.  Also add one contact with no values set (all empty)  and see what the server tell you about it